src/sim/V2/swig/python/simV2_wrap.cxx

```diff
@@ -3618,9 +3618,6 @@ SWIG_AsVal_unsigned_SS_char (PyObject * obj, unsigned char *val)
 #ifndef SWIG_isfinite
 # if defined(isfinite)
 #  define SWIG_isfinite(X) (isfinite(X))
-# elif defined __cplusplus && __cplusplus >= 201103L
-#  include <cmath>
-#  define SWIG_isfinite(X) (std::isfinite(X))
```

Comment:
Why was this removed?

Reply:
I have not idea, it is auto generated. I have restored it.